protocols/1551-universiat-hamburg/biofilm_assay.ot2.py

```diff
@@ -0,0 +1,93 @@
+from opentrons import labware, instruments, modules, robot
+
+metadata = {
+    'protocolName': 'CSV Plate Filling',
+    'author': 'Nick <protocols@opentrons.com>',
+    'source': 'Custom Protocol Request'
+}
+
+# create custom labware
+plate_name = '24-well-plate'
+if plate_name not in labware.list():
+    labware.create(
+        plate_name,
+        grid=(6, 4),
+        spacing=(20, 20),
+        diameter=16,
+        depth=14,
+        volume=1000
+    )
+
+# labware
+trough = labware.load('trough-12row', '3')
+tips300 = [labware.load('opentrons-tiprack-300ul', slot)
+           for slot in ['6', '9']]
+tubes_50 = labware.load('opentrons-tuberack-50ml', '11')
+
+# modules
+tempdeck = modules.load('tempdeck', '4')
+plate = labware.load('24-well-plate', '4', share=True)
+tempdeck.set_temperature(60)
+tempdeck.wait_for_temp()
+
+# pipette
+p300 = instruments.P300_Single(
+    mount='right',
+    tip_racks=tips300
+)
+
+# reagent setup
+water = trough.wells('A1')
+crystal_violet = trough.wells('A2')
+acetic_acid = trough.wells('A3')
+
+
+def run_custom_protocol():
+    robot.pause("Place the 24-well plate on the tempdeck. Wait 60 minutes "
```

Comment:
let's test on robot to make sure this works, and you can also `remove run_custom_protocol()`